web_app/hazard-reporting-map/pages/incidents/index.tsx

```diff
@@ -0,0 +1,371 @@
+import DataTable, { TDataTableColumn } from '@/components/DataTable';
+import Container from '@/components/layouts/Container';
+import { Badge } from '@/components/ui/badge';
+import { Button } from '@/components/ui/button';
+import { Checkbox } from '@/components/ui/checkbox';
+
+import { Input } from '@/components/ui/input';
+
+import ClientComponent from '@/components/ClientComponent';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from '@/components/ui/dropdown-menu';
+import data from '@/public/fake_incident_response.json';
+import {
+  ChevronDown,
+  ChevronLeft,
+  ChevronRight,
+  Pencil,
+  Filter,
+  Plus,
+  Search,
+  Trash,
+  Trash2,
+  Eye
+} from 'lucide-react';
+import { useState } from 'react';
+import ReactPaginate from 'react-paginate';
+
+export default function IncidentsLisPage() {
```

Comment:
We can rename the class to `IncidentsListPage`. I think there is a typo here.

Reply:
sure. done